hack/jenkins/update-jobs.sh

```diff
@@ -42,4 +42,4 @@ fi
 
 docker exec job-builder git checkout master
 docker exec job-builder git pull
-docker exec job-builder jenkins-jobs test ${config_dir}
+docker exec job-builder jenkins-jobs update ${config_dir}
```

Comment:
Okay, added that in. JJB doesn't have support for that plugin, so for now I have to use raw XML. The only other plugin that we regularly use that we'll have to use XML for is the slack notifier, although [this repo](https://github.com/asmundg/jenkins-jobs-slack) might be worth using for that.